### DISCUSSION TITLE
Please add 1.5 support

### DISCUSSION THREAD

**Discussion by jockisen** [lllyasviel/Fooocus#3467] [Ideas]
I've read that it's a big undertaking to give Fooocus more than a SDXL support. Just asking if 1.5 support could come? Fooocus is so fast even on slower GPUs so working with Hyper/Lightning solutions alongside not only SDXL would be awesome.

**Comment by mashb1t** [comment]
No, Fooocus supports SD1.5 refiners by using an latent interposer, but will not support SD1.5 as base models.
